syft/cataloging/relationships.go

```diff
@@ -0,0 +1,30 @@
+package cataloging
+
+type RelationshipsConfig struct {
+	FileOwnership                                 bool `yaml:"file-ownership" json:"file-ownership" mapstructure:"file-ownership"`
+	FileOwnershipOverlap                          bool `yaml:"file-ownership-overlap" json:"file-ownership-overlap" mapstructure:"file-ownership-overlap"`
+	ExcludeBinaryPackagesWithFileOwnershipOverlap bool `yaml:"exclude-binary-packages-with-file-ownership-overlap" json:"exclude-binary-packages-with-file-ownership-overlap" mapstructure:"exclude-binary-packages-with-file-ownership-overlap"`
```

Comment:
I think this name is super long, but it's clearer than the names above. It means, "If Syft finds a binary, say `java`, and also an RPM that installed `java`, it should only report the RPM," right?

Reply:
yeah, this name is terrible... but couldn't find a shorter name that wasn't potentially confusing. Side note: this is the option that is at odds between the Relationships and Packages config in the CLI/API levels .

Reply:
TODO: write and example in the docs